### ISSUE TITLE
JSON not serializable when using UUIDField

### ISSUE THREAD

**Issue by ubeerli** [yourlabs/django-autocomplete-light#781]
When using an UUIDField for the ID (PK) field, the ID is not directly serializable, since it is of type UUID. The function `get_result_value` in the class `BaseQuerySetView` in `views.py` should take this into account.
I suggest changing this:
```
def get_result_value(self, result):
    """Return the value of a result."""
    return result.pk
```
to `return six.text_type(result.pk)` as you do it for the label.

**Comment by jpic** [comment]
This makes sense to me, we'll need a regression test with the patch though.

**Comment by blag** [comment]
This is fixed in #781. The Travis tests that are currently failing in that PR failed when I just added a comment ([details](https://github.com/yourlabs/django-autocomplete-light/pull/799#discussion_r100224217)).
